### PR TITLE
H_COL2YAW ref fix and grammar pass

### PR DESCRIPTION
**Pull request by TunaLobster**

### CODE REVIEW

File: copter/source/docs/traditional-helicopter-parameter-list.rst [hunk 1]
```diff
@@ -34,11 +34,11 @@ These set the cyclic max, allow bench testing of blade angle setup, and setting
...
 ==================================
 
-These setup the configuration of the swashplate and collective travel.
+These set up the configuration of the swashplate and collective travel.
 
 - :ref:`H_SW_TYPE<H_SW_TYPE__AP_MotorsHeli_Single>`
-- :ref:`H_SW_H3_ENABLE<H_SW_H3_ENABLE__AP_MotorsHeli_Single>`  Never manually set this parameter!
-- ``H_COLYAW<H_COLYAW`` This parameter will be eliminated in versions after 4.3
+- :ref:`H_SW_H3_ENABLE<H_SW_H3_ENABLE__AP_MotorsHeli_Single>` Never manually set this parameter!
+- :ref:`H_COL2YAW<H_COLYAW>` This parameter will be eliminated in versions after 4.3
```
Comment (Hwurzburg) [inline copter/source/docs/traditional-helicopter-parameter-list.rst:41]: this is incorrect....please revert

File: copter/source/docs/traditional-helicopter-parameter-list.rst [hunk 2]
```diff
@@ -120,7 +120,7 @@ Parameters for internal throttle curve, if used.
 ArduPilot Provided Rotor Speed Governor
 =======================================
 
-Parameters for internal rotor speed governor, if used.  Internal throttle curve will need to be set up to use this mode.
+Parameters for internal rotor speed governor, if used. The internal throttle curve will need to be set up to use this mode.
```
Comment (Hwurzburg) [inline copter/source/docs/traditional-helicopter-parameter-list.rst:123]:
```suggestion
Parameters for internal rotor speed governor, if used. The transmitter's  throttle curve will need to be set up to use this mode. See the Setup section of :ref:`traditional-helicopter-internal-rsc-governor`.
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: copter/source/docs/traditional-helicopter-parameter-list.rst (hunk 1 above)
Comment (TunaLobster) [reply copter/source/docs/traditional-helicopter-parameter-list.rst:41]: Where should the new parameter be referenced then?

Comment (Hwurzburg) [reply copter/source/docs/traditional-helicopter-parameter-list.rst:41]: it will be when I get to it on the 4.4 list to add to the future wiki changes

**Comment by Hwurzburg** [comment]
thanks

**Pull request merged**